### PR TITLE
feature/LWF-3_setup-dark-theme

### PR DESCRIPTION
**Pull request by JeriRov**

### CODE REVIEW

**Review by borjom1 — APPROVED** [review]
LGTM

File: frontend/package-lock.json
Comment (milkanuch) [inline frontend/package-lock.json:1]: Are these changes needed?

File: frontend/src/components/ParallaxImage/ParallaxImage.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import Image, { ImageProps } from "next/image";
+import { ForwardedRef, forwardRef } from "react";
+
+export const ParallaxImage = forwardRef(
+  (props: ImageProps, ref: ForwardedRef<HTMLImageElement>) => {
+    return <Image width={500} height={500} ref={ref} {...props} />;
```
Comment (milkanuch) [inline frontend/src/components/ParallaxImage/ParallaxImage.tsx]: Make constants for width and height, please.
